pkg/config/services.go

```diff
@@ -512,6 +512,16 @@ var SupportedServices = serviceConfigs{
 			regexp.MustCompile(":application/(?P<Application>[^/]+)"),
 		},
 	},
+	{
+		Namespace: "AWS/KMS",
+		Alias:     "kms",
+		ResourceFilters: []*string{
+			aws.String("kms:key"),
+		},
+		DimensionRegexps: []*regexp.Regexp{
+			regexp.MustCompile(":key/(?P<KeyId>[^/]+)"),
+		},
+	},
```

Comment:
Can you please also add a test case for the associator?

Reply:
Added the test case.